mysql-test/main/auto_increment.test

```diff
@@ -433,3 +439,28 @@ SET @engine='MyISAM';
 --echo # End of 10.2 tests
 --echo #
 
+--echo #
+--echo # Start of 10.4 tests
```

Comment:
I don’t think that a "Start of" marker adds any value. The value of the "End of" marker is that it helps avoid merge conflicts.

Reply:
ack